states/team.py

```diff
@@ -1,5 +1,6 @@
 
 
 class Team:
+    """Function for Team Option"""
```

Comment:
See comments on `states/battle.py`

Reply:
Changed to `Displays team for player choosing Team state`

Reply:
Minor: make this more consistent with `states/battle.py`'s docstring:
`"""Game state for player battling"""`

Reply:
Changed to `Team state for player viewing team`